### PR TITLE
PLT-803: Support filter by status and health for appliances datasource.

### PR DESCRIPTION
**Pull request by nikchern**

### CODE REVIEW

**Review by SivaanandM — APPROVED** [review]

File: docs/data-sources/appliance.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
 page_title: "spectrocloud_appliance Data Source - terraform-provider-spectrocloud"
 subcategory: ""
 description: |-
-  
+  Provides details about a single appliance.
```
Comment (karl-cardenas-coding) [inline docs/data-sources/appliance.md:6]: This needs more information @nikchern. Explain what an appliance is.

File: docs/data-sources/appliance.md [hunk 2]
```diff
@@ -27,14 +27,14 @@ output "same" {
...
-- `name` (String)
-
-### Optional
-
-- `tags` (Map of String)
+- `name` (String) The name of the appliance.
 
 ### Read-Only
 
+- `architecture` (String) The architecture of the appliance. Possible values are: 'amd64', 'arm64'.
```
Comment (karl-cardenas-coding) [inline docs/data-sources/appliance.md:34]:
```suggestion
- `architecture` (String) The architecture of the appliance. Allowed values are: 'amd64', and 'arm64'.
```

File: docs/data-sources/appliance.md [hunk 3]
```diff
@@ -27,14 +27,14 @@ output "same" {
...
+- `health` (String) The health of the appliance. Possible values are: 'healthy', 'unhealthy'.
```
Comment (karl-cardenas-coding) [inline docs/data-sources/appliance.md:35]:
```suggestion
- `health` (String) The health of the appliance. Allowed values are: 'healthy', and 'unhealthy'.
```

File: docs/data-sources/appliance.md [hunk 4]
```diff
@@ -27,14 +27,14 @@ output "same" {
...
 - `id` (String) The ID of this resource.
+- `status` (String) The status of the appliance. Possible values are: 'ready', 'in-use', 'unpaired'.
```
Comment (karl-cardenas-coding) [inline docs/data-sources/appliance.md:37]:
```suggestion
- `status` (String) The status of the appliance. Allowed values are: 'ready', 'in-use', and 'unpaired'.
```

**Review by karl-cardenas-coding — COMMENTED** [review]
Some suggestions

File: docs/data-sources/appliance.md (hunk 1 above)
Comment (nikchern) [reply docs/data-sources/appliance.md:6]: Sure will do @karl-cardenas-coding

**Pull request merged**